Smart Village App/views/stream/widgets/wallStreamFilterNavigation.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
as a test for referencing the original file: https://github.com/humhub/humhub/blob/v1.9/protected/humhub/modules/stream/widgets/views/wallStreamFilterNavigation.php

Reply:
all right nice, it is quickly recognizable, that the only change is the new <script> part at the end of the file.

Reply:
Hi @donni106 
Yes, the only is we add <script> tag to hide the post section